doc/api/module.md

```diff
@@ -931,6 +931,46 @@ added:
 `path` is the resolved path for the file for which a corresponding source map
 should be fetched.
 
+### `module.prepareStackTrace(error, trace)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `error` {Object}
+* `trace` {Object\[]}
+* Returns: {string} Returns formatted stack trace.
+
+This is the default implementation Node.js stack trace formatter. When source
+map support is enabled, this function will attempt to map the stack trace to
```

Comment:
We should update the section in `cli.md` https://github.com/nodejs/node/blob/02db7fc2cd3cdfc2b097a983449b33f54735df4b/doc/api/cli.md#L526-L527